tilavarauspalvelu/api/graphql/types/equipment/types.py

```diff
@@ -16,3 +16,15 @@ class Meta:
         ]
         filterset_class = EquipmentFilterSet
         permission_classes = [EquipmentPermission]
+
+
+class EquipmentAllNode(DjangoNode):
```

Comment:
I would put a comment to these nodes warning about possible misuse (i.e. don't expose any relations later)